samples/core/Querying/Querying/RawSQL/Sample.cs

```diff
@@ -27,7 +27,7 @@ public static void Run()
                 var user = "johndoe";
 
                 var blogs = context.Blogs
-                    .FromSql("EXECUTE dbo.GetMostPopularBlogsForUser {0}", user)
+                    .FromSqlInterpolated("EXECUTE dbo.GetMostPopularBlogsForUser {0}", user)
```

Comment:
This call needs to change if we want to turn it into an example for FromSqlInterpolated.